src/components/PostForm.tsx

```diff
@@ -0,0 +1,120 @@
+import { h, Component } from 'preact';
+import { Button } from '@mui/material';
+import { newPublicPost, getCurrentAuthor } from '../utils/apiCalls';
+
+type Props = {  };
```

Comment:
you don't seem to be using this function anywhere in the code, can you remove it then?